app/routes/users/components/UserSettingsOAuth2.css

```diff
@@ -5,14 +5,3 @@
 .copied {
   color: var(--color-green-6);
 }
```

Comment:
While you're at it, you could remove this and add `success={copied}` to the icon in `app/routes/users/components/UserSettingsOAuth2.tsx`.